manta-trusted-setup/src/groth16/ceremony/mod.rs

```diff
@@ -194,8 +194,8 @@ where
         match self {
             Self::NotRegistered => write!(
                 f,
-                "Registry update is taking longer than expected. \
-                 Please make sure you have submitted your registration form and try again later.",
+                "Please make sure you have submitted your registration form. \
+                 Check whether the ceremony has begun at https://ceremony.manta.network.",
```

Comment:
I take people are getting this error because the box is running with our test server. Because otherwise redirecting them to ceremony.manta.network makes more sense when network errors happen.